### ISSUE TITLE
What does parameter "use_tef" means?

### ISSUE THREAD

**Issue by EasonXiao-888** [wjun0830/QD-DETR#22]
Hello,
Thanks for the great work.I would like to ask what this parameter"use_tef" represents. Why concat "tef" with  "model_inputs["video_feat"] " if "use_tef" is set True. The code on line 97 of file "QD-DETR/qd_detr/start_end_dataset.py" shows below:
if self.use_tef:
            tef_st = torch.arange(0, ctx_l, 1.0) / ctx_l
            tef_ed = tef_st + 1.0 / ctx_l
            tef = torch.stack([tef_st, tef_ed], dim=1)  # (Lv, 2)
            if self.use_video:
                model_inputs["video_feat"] = torch.cat(
                    [model_inputs["video_feat"], tef], dim=1)  # (Lv, Dv+2)
            else:
                model_inputs["video_feat"] = tef
best,
Eason

**Comment by wjun0830** [comment]
Hello. Thanks for your interest.
You can find your answer [here](https://github.com/jayleicn/moment_detr/issues/7).

**Comment by EasonXiao-888** [comment]
I got it. Thanks very much
